VocaDbModel/Database/Queries/SongQueries.cs

```diff
@@ -821,8 +821,10 @@ public async Task<NewSongCheckResultContract> FindDuplicates(string[] anyName, s
 
 				var matches = pvMatches.Union(nameMatches)
 					.Select(s => new DuplicateEntryResultContract<SongMatchProperty>(new EntryRefWithCommonPropertiesContract(s.song, PermissionContext.LanguagePreference), s.property))
+					.GroupBy(s => s.Entry.Id)
+					.Select(s => s.First())

```

Comment:
We also have the [Distinct](https://github.com/VocaDB/vocadb/blob/4fa48be2f8e0b05ea6f81e5f8ce629a489249de1/VocaDbModel/Helpers/EnumerableExtensions.cs#L18) extension method with a property selector. See also https://github.com/VocaDB/vocadb/pull/727#discussion_r557245029 and https://github.com/VocaDB/vocadb/commit/a321697c90308a919ff5d1fb5a6ba56a7d7041a1.